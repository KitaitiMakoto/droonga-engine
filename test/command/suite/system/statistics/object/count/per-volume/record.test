#@require-catalog-version 2
#@disable-logging
{
  "type": "add",
  "dataset": "Default",
  "body": {
    "table": "Users",
    "key": "Adam"
  }
}
#@enable-logging
{
  "type": "system.statistics.object.count.per-volume",
  "dataset": "Default",
  "body": {
    "output": [
      "tables",
      "columns",
      "records",
      "total"
    ]
  }
}
