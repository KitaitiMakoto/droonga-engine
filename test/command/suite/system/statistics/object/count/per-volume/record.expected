{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "system.statistics.object.count.per-volume.result",
  "body": {
    "127.0.0.1:42300/droonga.000": {
      "tables":  1,
      "columns": 0,
      "records": 1,
      "total":   2
    },
    "127.0.0.1:42300/droonga.010": {
      "tables":  1,
      "columns": 0,
      "records": 1,
      "total":   2
    }
  }
}
